src/appleseed.studio/mainwindow/project/disneymateriallayerui.cpp

```diff
@@ -185,19 +186,19 @@ void DisneyMaterialLayerUI::create_input_widgets(const Dictionary& values)
             im.strings().exist("default") ? im.get<string>("default") :
             "");
 
-        auto_ptr<IInputWidgetProxy> widget_proxy =
+        unique_ptr<IInputWidgetProxy> widget_proxy =
             input_type == "colormap" ?
                 im.dictionaries().exist("entity_types") &&
                 im.dictionaries().get("entity_types").strings().exist("color")
                     ? create_color_input_widgets(im)
                     : create_colormap_input_widgets(im) :
             input_type == "text" ? create_text_input_widgets(im) :
-            auto_ptr<IInputWidgetProxy>(0);
+            unique_ptr<IInputWidgetProxy>(nullptr);
 
         assert(widget_proxy.get());
         connect(widget_proxy.get(), SIGNAL(signal_changed()), SIGNAL(signal_apply()));
 
-        m_widget_proxies.insert(input_name, widget_proxy);
+        m_widget_proxies.insert(input_name, std::move(widget_proxy));
```

Comment:
`std::` qualifier redundant? (In many other places as well.)

Reply:
These qualifiers are needed for gcc 4.8 to compile the code.
Some libraries define move methods and functions, for example Qt and the compiler does not pick std::move.